src/lib.zig

```diff
@@ -3171,6 +3189,19 @@ pub const Lua = struct {
                     },
                 }
             },
+            .Array, .Vector => {
+                const child = std.meta.Child(T);
+                const arr_len = switch (@typeInfo(T)) {
+                    inline else => |i| i.len,
+                };
+                var result: T = undefined;
+                for (0..arr_len) |i| {
+                    _ = lua.getIndex(index, @intCast(i + 1));
```

Comment:
getIndex is only available in Lua 5.3 and 5.4.

Unless you need the metatable access support, it would be better to use `rawGetIndex` here instead.

Reply:
I made it compatible with older versions. Metatable access may be useful for userdata.